system/include/emscripten/em_asm.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __em_asm_h__
```

Comment:
Actually none of the other headers in the `system/include/emscripten` folder seem to have them. @kripken is that a thing?